python/nano/src/bigdl/nano/pytorch/onnx/onnxrt_inference.py

```diff
@@ -174,7 +175,9 @@ def bind_onnxrt_methods(pl_model: LightningModule):
     # check conflicts
     for component in ONNXRT_BINDED_COMPONENTS:
         if component in dir(pl_model):
-            warnings.warn(f"{component} method/property will be replaced.")
+            warnings.warn(f"{component} method/property will be replaced. You may rename your",
+                          " customized attributes or methods and call `Trainer.compile again `",
+                          "to avoid being overwrite.")
```

Comment:
do we need to report warning for `on_fit_start`?

Reply:
yes, since `on_fit_start` is now overwritten rather than use a decorator to change it. We need to report the warning. I drop the decorator because that the decorator implementation has no way to support multi-processing training (which needs to pickle the model)

Reply:
But can the user do anything about it? And we just append something to the original `on_fit_start`, instead of replacing it, yes?

Reply:
we "should" append something, still now we replace it (to support multi-processing). I may find some solution to append and support the multi-processing. (don't have any good idea right now)

Reply:
well they may do some thing in other hook rather than `on_fit_start`.

Reply:
and of course, that's not a good way to do it.